pkg/sqlstore/impl/system_store_instrumented.go

```diff
@@ -208,6 +209,7 @@ func (s *InstrumentedSystemStore) Begin(ctx context.Context) (*sql.Tx, error) {
 func (s *InstrumentedSystemStore) GetReceipt(
 	ctx context.Context,
 	txnHash string) (eventprocessor.Receipt, bool, error) {
+	log.Debug().Str("hash", txnHash).Msg("call GetReceipt")
```

Comment:
This is unrelated to the new rpc endpoint, but being able to see the `tableland_getReceipt` rpc call in DEBUG mode will be convenient when we implement  https://www.notion.so/textile/Handle-receipt-confirmations-in-JS-SDK-82da86b6184f4472a42d697b5ef91f9f